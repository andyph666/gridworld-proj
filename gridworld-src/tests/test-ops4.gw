/* not sure use && || or "and,or", both occurs in the LRM

if(0&&1) {print("yes1");}
if(0||1) {print("yes2");}
if(0and1) {print("yes3");}
if(0or1) {print("yes4");}
if(True&&False) {print("yes5");}
if(True||False) {print("yes6");}
