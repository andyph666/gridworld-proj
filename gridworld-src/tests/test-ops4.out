yes2
yes4
yes6